sql/core/src/test/scala/org/apache/spark/sql/DataFrameWriterV2Suite.scala

```diff
@@ -100,86 +100,6 @@ class DataFrameWriterV2Suite extends QueryTest with SharedSparkSession with Befo
     assert(v2.catalog.exists(_ == catalogPlugin))
   }
 
-  case class FakeV2WriteCommand(table: NamedRelation, query: LogicalPlan) extends V2WriteCommand
-
-  test("SPARK-33136 output resolved on complex types for V2 write commands") {
```

Comment:
This test doesn't use `DataFrameWriterV2` at all. I moved it to `DataSourceV2AnalysisBaseSuite`, so that we don't need the `FakeV2WriteCommand`.